pkg/get/tools.go

```diff
@@ -669,6 +669,38 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 			https://releases.hashicorp.com/{{.Name}}/{{.Version}}/{{.Name}}_{{.Version}}_{{$os}}_{{$arch}}.zip`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "gruntwork-io",
+			Repo:        "terragrunt",
+			Name:        "terragrunt",
+			Version:     "v0.37.1",
```

Comment:
It doesn't necessary make sense to pin the version while working with GitHub releases, arkade can fetch the API and get the latest.
Please remove `Version`.

Reply:
will do thanks!